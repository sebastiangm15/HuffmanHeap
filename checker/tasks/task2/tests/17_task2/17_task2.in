15
3 IBQ59
3 OAR33
7 ODH38
11 YQX18
14 LKK68
34 IJA29
46 UFP91
58 FXS15
64 GPQ42
72 IBU28
79 YTQ74
87 KDS34
93 NFV74
95 YIU11
98 RUW4
4
000111000110001100000
11011100
11100101111
1100010001
