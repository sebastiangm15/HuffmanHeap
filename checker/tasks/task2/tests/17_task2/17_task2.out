IBQ59 NFV74 YQX18 IJA29 
FXS15 UFP91 
GPQ42 KDS34 IBU28 
UFP91 KDS34 YTQ74 
